events/klasaReady.js

```diff
@@ -0,0 +1,27 @@
+const { Event } = require('klasa'); // eslint-disable-line no-unused-vars
+
+module.exports = class extends Event {
+
+    constructor(...args) {
+        super(...args, { once: true });
+    }
+
+    async run() {
+        //
+    }
+
+    async init() {
+        // Folders
+        if (!this.client.gateways.guilds.schema.has('channels')) await this.client.gateways.guilds.schema.add('channels', {});
```

Comment:
All these should be in the related files, such as announcement for roles.announcementRole

Reply:
I won't be doing this change as it would lead to slight code duplication (checking if the schema has channels) for all related files.

I feel doing this check in the klasaReady event is the best course of action.